modules/op-installing-tkn-on-linux-using-rpm.adoc

```diff
@@ -47,7 +47,7 @@ For {op-system-base-full} version 8, you can install the {pipelines-title} CLI (
 +
 [source,terminal]
 ----
-# subscription-manager repos --enable="pipelines-1.3-for-rhel-8-x86_64-rpms"
+# subscription-manager repos --enable="pipelines-1.4-for-rhel-8-x86_64-rpms"
```

Comment:
@VeereshAradhya can you confirm that the repo is correct (i.e. there was no change in naming conventions for GA)

Reply:
@VeereshAradhya do we have a confirmation on this?

Reply:
yes.. verified.. looks good